### PR TITLE
add Anikcb

### LINKED ISSUE #654: Anik Chakraborty
image

### PR DESCRIPTION
**Pull request by Anikcb**
resolve #654

### CODE REVIEW

**Comment by CodingSpecies** [comment]
@schmelto, I think this can be merged.

**Review by KukretiShubham — APPROVED** [review]

**Comment by KukretiShubham** [comment]
LGTM 🚀

File: data.json [hunk 1]
```diff
@@ -1270,5 +1270,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/51391473/129485691-4a1249b6-3203-4f15-ac8c-654a67d50ee4.png",
     "issueNumber": 648
-  }
+  },
+  {
+    "name": "Anik Chakraborty",
+    "githubUsername": "Anikcb",
+    "imageUrl": "https://user-images.githubusercontent.com/47493387/130018556-ee4c3974-fcb7-4760-b988-00ee0a0c8947.png",
+    "issueNumber": 654
+}
```
Comment (schmelto) [inline data.json:1279]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
> think this can be merged.

One higlight from my side. If you want to review a PR you cann comment and approve/comment"request changes in this section here
image

with this your contribution graph will be changed to code review
image

**Comment by CodingSpecies** [comment]
@schmelto, thank you for that tip! :)

**Pull request closed without merging**